resolwe_bio/processes/amplicon/bwa_trim.yml

```diff
@@ -0,0 +1,180 @@
+# ==========================================
+# Align reads and trimm resulting SAM output
+# ==========================================
+---
+
+- slug: align-bwa-trim
```

Comment:
Could this be an option in the current bwa aligner? For instance, trim primers if master file given.